exercises/J-array-get-set/exercises2.js

```diff
@@ -7,7 +7,7 @@
 */
 
 var numbers = [1, 2, 3]; // Don't change this array literal declaration
-
+numbers[3] = 4;
 /* 
```

Comment:
There is a red cross on the top of this commit. I think may be you need to assign numbers[0] = 1; as asked in the question.

Reply:
Isn't the first value in the array already the number 1. Do I need to change the number 1 with number 1? Why? I don’t get it 😃

Reply:
yes i agree. the question was bit silly. But that's what the question asks us to do. may be that will remove the red cross on your file.